src/builder.ts

```diff
@@ -44,8 +44,13 @@ export async function buildJDK(
   //pre-install dependencies
   await installDependencies(javaToBuild, impl)
   let jdkBootDir = ''
-  const bootJDKVersion = getBootJdkVersion(javaToBuild)
-  
+
+  let tempBootJDKVersion = getBootJdkVersion(javaToBuild)
+  if (parseInt(tempBootJDKVersion) == 10) {
+    tempBootJDKVersion = '11'
+  }
+  const bootJDKVersion = tempBootJDKVersion
+
```

Comment:
Probably we don't need an extra variable. 

```
let bootJDKVersion = getBootJdkVersion(javaToBuild)
if (parseInt(tempBootJDKVersion) == 10) {
   bootJDKVersion = '11'
  }

```

Reply:
Okie dokie. Change made. New run: https://github.com/adamfarley/build-jdk/actions/runs/903490320